src/current/cockroachcloud/network-authorization.md

```diff
@@ -15,12 +15,19 @@ You can authorize network access to your cluster by:
 
 - [Adding an authorized range of public IP addresses](#ip-allowlisting).
 - Setting up [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering](#vpc-peering) or [Amazon Web Service (AWS) PrivateLink](#aws-privatelink) for your cluster (Dedicated clusters only). Access via [GCP VPC peering](#vpc-peering) or [AWS PrivateLink](#aws-privatelink) avoids traversing the public network, and therefore offers several advantages:
-
     - Enhanced network security (no access through public IPs i.e. no transit over public networks).
     - Direct connection from application deployment that do not have static public IPs
-    - Reduced lower network latency.
+    - Reduced network latency.
     - This option can also be helpful if you need more than the current maximum limit of authorized networks per cluster.
 
+**Prerequisite**: {% include cockroachcloud/cluster-operator-prereq.md %}
+
+{{site.data.alerts.callout_success}}
+You should use PrivateLink or VPC peering if you need to allowlist more IP addresses than allowed by the maximum (20 for Dedicated clusters and 50 for Serverless), if your servers’ IP addresses are not static, or if you want to limit your cluster's exposure to the public internet.
```

Comment:
I don't know how this would look in this callout box... (and this in the intended meaning!), but what about a structure like this:

> You should use PrivateLink or VPC peering if:

> - You need to allowlist more IP addresses than allowed by the maximum (20 for Dedicated clusters and 50 for Serverless).
> - Your servers’ IP addresses are not static.
> - You want to limit your cluster's exposure to the public internet.

Reply:
thanks!